plugins/rss/init.js

```diff
@@ -1148,17 +1153,25 @@ rTorrentStub.prototype.getrssdetails = function()
 
 rTorrentStub.prototype.setrsssettings = function()
 {
-	this.rssCommon("mode=setsettings&interval="+this.ss[0]+"&delayerrui="+this.ss[1]);
+	this.rssCommon("mode=setsettings&" + ['interval', 'delayerrui', 'relaxsanitization'].map((n,i) => `${n}=${this.ss[i]}`).join('&'));
 }
 
 rTorrentStub.prototype.getrssdetailsResponse = function(data)
 {
-    	const doc = new DOMParser().parseFromString(String(data), 'text/html');
-    	var s = new Sanitize(Sanitize.Config.RESTRICTED);
-    	$("#rsslayout").html(s.clean_node(doc.body));
-    	return(false);
-}
-
+	const cfg = Sanitize.Config[
+		Sanitize.Config.Levels[theWebUI.rssRelaxSanitization] || "RESTRICTED"
+	];
+	const s = new Sanitize({
+		...cfg,
+		elements: [...cfg.elements, "ins", "details", "summary"],
+		transformers: [bbclassTransform],
+	});
+	const rawHTML = String(data);
+	const dirtyHTML = theWebUI.mapBBCodeToHTML(rawHTML);
```

Comment:
This is from the other pull request. We need changes tested and we can't be stringing pull requests like this.

Reply:
Sorry, I only fully test all changes at once. This is what the jest tests should have caught.